### PR TITLE
Added the version 2.8.1 changelog

### PR DESCRIPTION
**Pull request by jnidzwetzki**
# Description

Update the release notes with the 2.8.1 changelog

# Links

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -20,6 +20,40 @@ Want to stay up-to-date with new releases? Subscribe to get
 *   1 step continuous aggregate policy management
 *   Migrate continuous aggregates to the new format
 
+## 2.8.1 (2022-10-06)
+
+This release is a patch release. We recommend that you upgrade at the
+next available opportunity.
+
+**Bugfixes**
```
Comment (iroussos) [inline timescaledb/overview/release-notes/index.md:28]:
```suggestion
**Bug fixes**
```

**Review by iroussos — APPROVED** [review]
Just a minor typo suggestion on my side

File: timescaledb/overview/release-notes/index.md (hunk 1 above)
Comment (jnidzwetzki) [reply timescaledb/overview/release-notes/index.md]: @iroussos Thanks for the hint. It is changed in the current version of the PR.

**Review by Loquacity — APPROVED** [review]

**Review by jfjoly — APPROVED** [review]
LGTM

**Pull request merged**